APE14.rst

```diff
@@ -190,10 +201,11 @@ low-level API recommends:
             The bounds (in pixel coordinates) inside which the WCS is defined,
             as a list with ``pixel_n_dim`` ``(min, max)`` tuples (optional).
 
-            WCS solutions are sometimes only guaranteed to be accurate within a
-            certain range of pixel values, for example when definining a WCS
-            that includes fitted distortions. This is an optional property, and
-            it should return `None` if a shape is not known or relevant.
+            The bounds should be given in ``[(xmin, xmax), (ymin, ymax)]``
+            order. WCS solutions are sometimes only guaranteed to be accurate
+            within a certain range of pixel values, for example when defining a
+            WCS that includes fitted distortions. This is an optional property,
+            and it should return `None` if a shape is not known or relevant.
```

Comment:
Shouldn't _"...if a shape is not known or relevant."_ be something like _"... if a shape is neither known nor relevant."_? The way this is currently written is confusing to me (but it just could be my English): does **not** apply to **known** and not to **relevant** or does **not** apply to _both_ **known** _and_ **relevant**?

Same goes for `numpy_shape`, `pixel_shape`.

Reply:
Fixed